flink-ai-flow/ai_flow/workflow/job.py

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+from typing import Text
+from ai_flow.graph.node import BaseNode
+from ai_flow.workflow.job_config import JobConfig
+
+
+class Job(BaseNode):
+    """
+    A Job contains the common information of a ai flow job. Users can implement custom jobs by adding other
```

Comment:
It took me quite a while to read this doc yet still do not understand what exactly this means....